bin/create-release-binaries.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+PLUGIN_NAME="cf-service-connect"
+
+GOOS=darwin GOARCH=amd64 go build -o ${PLUGIN_NAME}_darwin_amd64
```

Comment:
Could we also have M1's covered by:
GOOS=darwin GOARCH=arm64 go build -o ${PLUGIN_NAME}_darwin_arm64